Core.ApplicationServices/Organizations/OrganizationUnitService.cs

```diff
@@ -224,6 +243,14 @@ public Maybe<OperationError> TransferRegistrations(Guid organizationUuid, Guid u
                 );
         }
 
+        private UnitAccessRights GetAccessRights(OrganizationUnit unit)
+        {
+            var canBeModified = _authorizationContext.AllowModify(unit);
+            var canBeDeleted = _authorizationContext.AllowDelete(unit);
+
+            return unit.GetAccessRights(isUserAllowedToRead: true, canBeModified, canBeDeleted);
```

Comment:
I think we need to talk about this approach because "access" is not part of the domain...